src/style.css

```
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.wrapper {
+  margin: 20px auto;
+  position: relative;
+}
+
+.form-input {
+  outline: none;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search-input::after {
+  content: "";
+  background: url(../src/images/Search.svg) no-repeat center center;
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  display: block;
+  background-size: contain;
+  top: 25px;
+  left: 26px;
+}
+
+.form-input-small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search-input input:hover {
```

You should apply pseudoclass for class, not default tag
Please fix all cases
```suggestion
.form-input:hover {
```